trulens_eval/examples/expositional/models/hugging__Face_.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>Vectra -&gt; Vectara?</p><p> </p><p>Please remove the stray "or."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/950/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>